lms/static/sass/views/_teams.scss

```diff
@@ -1,3 +1,51 @@
+@mixin tooltip-hover-style ($margin-top) {
```

Comment:
I'm going to check-in with the UX team around existing patterns for this, since I'm seeing a couple in existence in our platform but most seem old / outdated. Thanks for making this a mixin!